tests/sentry/web/frontend/test_vercel_extension_configuration.py

```diff
@@ -47,6 +48,7 @@ def setUp(self):
         }
 
     @responses.activate
+    @with_feature("organizations:integrations-deployment")
```

Comment:
@snigdhas this was actually a real bug that was masked by adding `with_feature` since it's not actually registered.

Reply:
I'd like to have a follow-up here where we change how our feature mock library works to catch this kind of thing.